packages/suite-build/package.json

```diff
@@ -19,10 +19,11 @@
         "dev:desktop": "yarn run desktop",
         "build:desktop": "NODE_ENV=production yarn run desktop",
         "lint": "eslint '**/*.{ts,tsx,js}'",
-        "type-check": "tsc --project tsconfig.json",
+        "type-check": "tsc --build tsconfig.json",
         "type-check:watch": "yarn type-check -- --watch"
     },
     "dependencies": {
+        "@trezor/suite": "1.0.0",
```

Comment:
is it required?

Reply:
it is, there is package.json from suite imported somewhere in this package